lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers_test.go

```diff
@@ -1044,6 +1044,166 @@ func TestGetSubscriber(t *testing.T) {
 	tests.RunUnitTest(t, e, tc)
 }
 
+func TestGetSubscriberByExactIMSI(t *testing.T) {
+	configuratorTestInit.StartTestService(t)
+	deviceTestInit.StartTestService(t)
+	stateTestInit.StartTestService(t)
+	err := configurator.CreateNetwork(configurator.Network{ID: "n1"}, serdes.Network)
+	assert.NoError(t, err)
+
+	e := echo.New()
+	testURLRoot := "/magma/v1/lte/:network_id/subscribers/:subscriber_id"
+	handlers := handlers.GetHandlers()
+	getSubscriber := tests.GetHandlerByPathAndMethod(t, handlers, testURLRoot, obsidian.GET).HandlerFunc
+
+	// preseed 2 apns
```

Comment:
Only need to create 1 apn here
```suggestion
	// Pre: create an APN
```

Reply:
The current version of code makes use of the `newMutableSubscriber` helper function, which requires two APNs and is used by other tests as well. For simplicity's sake, will probably stick to having two APNs